src/main/java/pro/sky/stream_api_optional_2_8/exceptions/ArrayIsFullException.java

```diff
@@ -5,8 +5,12 @@
 
 @ResponseStatus(HttpStatus.BAD_REQUEST)
 public class ArrayIsFullException extends ResponseException {
-    public ArrayIsFullException(String message) {
-        super(message);
+    public ArrayIsFullException(String message, HttpStatus status) {
+        super(message, status);
     }
 
+    @Override
+    public HttpStatus getStatus() {
+        return super.getStatus();
+    }
```

Comment:
можно не переопределять тогда, раз внутри ты всё равно вызываешь родительскую реализацию. Просто удали этот метод и всё будет продолжать работать. Когда я писал про переопределение, то имел ввиду вот это:
```suggestion
    @Override
    public HttpStatus getStatus() {
        return HttpStatus.BAD_REQUEST;
    }
```
и тогда через конструктор можно не передавать. И над классом убрать аннотацию, она бесполезная, т.к. ты всё равно handler-ом обрабатываешь исключения